README.md

```diff
@@ -1043,6 +1043,8 @@ See also [About Rust’s Machine Learning Community](https://medium.com/@autumn_
   * [ZooKeeper](https://zookeeper.apache.org/)
     * [bonifaido/rust-zookeeper](https://github.com/bonifaido/rust-zookeeper) [[zookeeper](https://crates.io/crates/zookeeper)] — A client library for Apache ZooKeeper. [![build badge](https://api.travis-ci.org/bonifaido/rust-zookeeper.svg?branch=master)](https://travis-ci.org/bonifaido/rust-zookeeper)
     * [krojew/rust-zookeeper](https://github.com/krojew/rust-zookeeper) [[zookeeper-async](https://crates.io/crates/zookeeper-async)] - Async Zookeeper client written 100% in Rust, based on tokio.  ![build status](https://github.com/krojew/rust-zookeeper/actions/workflows/rust.yml/badge.svg)
+  * [PoloDB](https://www.polodb.org/)
+    * [vincentdchan/PoloDB](https://github.com/vincentdchan/PoloDB) - An embedded JSON-based database has API similar to MongoDB.
```

Comment:
Try to add the build status

besides that, I checked and have more than 50 stars
![image](https://user-images.githubusercontent.com/36519478/212012344-a6a7583b-d87c-4751-bc7f-601055231232.png)

Reply:
Build status added 🍻